dex-it/src/test/scala/com/wavesplatform/it/sync/PostgresHistoryDatabaseTestSuite.scala

```diff
@@ -210,6 +212,17 @@ class PostgresHistoryDatabaseTestSuite extends MatcherSuiteBase with HasPostgres
     ctx.run(querySchema[EventRecord]("events").delete)
   }
 
+  "Node should provide its name to postgres" in {
+
+    val buyOrder = mkOrder(bob, wavesBtcPair, BUY, 270477189L, 28259L)
+    placeAndAwaitAtDex(buyOrder)
```

Comment:
can we get app name from connection without placing an order?

Reply:
No. When I ran app, it didn't connect to db at start, only when I tried to place an order from code. Tried to remove order from this test, got the same behavior: test's failed because no such connection was found.